koku/masu/database/trino_sql/gcp/openshift/gcp_openshift_daily.sql

```diff
@@ -0,0 +1,151 @@
+-- Now create our proper table if it does not exist
+CREATE TABLE IF NOT EXISTS hive.{{schema | sqlsafe}}.gcp_openshift_daily
```

Comment:
How does this work with an existing customer that has a `gcp_openshift_daily` table that is created with an external location? We are also changing the column order/names for some of this which might cause us some issues. For example, locally `show create table gcp_openshift_daily` shows the following:
```
 WITH (                                                                                  
    external_location = 's3a://koku-bucket/data/parquet/daily/org1234567/GCP/openshift', 
    format = 'PARQUET',                                                                  
    partitioned_by = ARRAY['source','year','month','day']                                
 )
```

Reply:
This stuff _should_ be ok when we drop the current gcp_openshift_daily table. On the next code run we will create a new table with all the correct/relevant columns.